downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -84,3 +129,177 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
+        xml x = xml `<e/><f/>`;
+        xml x1 = x.<e>; // which returned empty xml now returns <e/>.
+        xmlns "http://example.com/";
+    }
+    ```
+
+- A bug which resulted in XML navigation failing to return elements with namespaces different from the default one when using `<*>` to access XML element children has been fixed.
+
+    ```ballerina
+    public function main() {
+        xmlns "foo";
+        xmlns "bar" as k;
+        xml x1 = xml `<item><k:child>C</k:child><child2>D</child2></item>`;
+        xml x2 = x1/<*>; // used to return `<child2 xmlns="foo">D</child2>`, now returns `<k:child xmlns="foo" xmlns:k="bar">C</k:child><child2 xmlns="foo">D</child2>`
+    }
+    ```
+
+- A bug which resulted in XML namespace URI being empty when a constant is used for XML namespace URI in the XML namespace definition has been fixed.
+
+    ```ballerina
+    const URI = "http://ballerina.com/";
+    xmlns URI as ns0;
+
+    public function main() {
+        string s = ns0:foo; // used to result in "{}foo" which now results in {http://ballerina.com/}foo
+    }
+    ```
+
+- A bug which resulted in a compiler crash or a null value for a variable of a binding pattern which is used within closure has been fixed.
+
+    ```ballerina
+    function f1() {
+        string[] values = ["a", "b", "c"];
+        map<int> data = {
+            a: 1
+        };
+        foreach var [k, _] in data.entries() {
+            boolean b = <any> k is (); // which resulted true since k is used in a closure, now returns false. k is now not nil.
+            _ = values.filter(item => item == k);
+        }
+    }
+    ```
+
+- A bug which resulted in empty XML results in XML navigation when the navigation name pattern contains escape characters has been fixed.
+
+    ```ballerina
+    public function main() {
+        xml d = xml `
+        <person>
+            <name>John</name>
+            <home-address>some address</home-address>
+        </person>`;
+
+        xml x1 = d/<home\-address>; // Used to produce an empty output. Now produce <home-address>some address</home-address>
+    }
+    ```
+
+- A bug which resulted in no compilation error on missing required fields in the select clause has been fixed.
+
+    ```ballerina
+    type Foo record {|
+        int i;
+        string x;
+    |};
+
+    type Bar record {|
+        int j;
+        string y;
+    |};
+
+    type Baz record {|
+        Foo[] fooArr;
+    |};
+
+    function fn(Baz baz) {
+        int|Bar[] _ = from var foo in baz.fooArr
+            select { // used to give no compilation error. Now gives a compilation error. missing non-defaultable required record field 'y'
+                j: foo.i
+                // y: foo.x
+            };
+    }
+    ```
+
+- A bug which resulted in swapping the top most comments above imports while formatting is fixed. This fix results in preserving newlines within comment blocks above imports.
+
+    - Before formatting
+
+    ```ballerina
```

Comment:
Can indent, right?

Reply:
It doesn't seem to be possible to indent the one single level list in an .md file. (when indented it is considered as a code.)